modulefiles/ufs_hercules.intel.lua

```diff
@@ -2,7 +2,7 @@ help([[
 loads UFS Model prerequisites for Hercules/Intel
 ]])
 
-prepend_path("MODULEPATH", "/work/noaa/epic-ps/role-epic-ps/spack-stack/spack-stack-1.4.0-hercules/envs/unified-env-v2/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/work/noaa/epic/role-epic/spack-stack/spack-stack-1.4.1-hercules/envs/unified-env-v2/install/modulefiles/Core")
```

Comment:
sorry...one more:

there are issues with gcc 12 on hercules, so neither the intel or gnu stacks are completed yet, but the path will be:

`/work/noaa/epic/role-epic/spack-stack/hercules/spack-stack-1.4.1/envs/unified-env/install/modulefiles/Core`

Reply:
done